resources/v1.26.2/charts/istiod/files/injection-template.yaml

```diff
@@ -95,7 +94,7 @@ spec:
     - "-z"
     - {{ .MeshConfig.ProxyInboundListenPort | default "15006" | quote }}
     - "-u"
-    - {{ if $tproxy }} "1337" {{ else }} {{ .ProxyUID | default "1337" | quote }} {{ end }}
+    - {{ .ProxyUID | default "1337" | quote }}
```

Comment:
Do you know where is this change coming from? I'm aware only about changes for `trustBundleName`